hosts_example.yml

```diff
@@ -118,6 +118,17 @@ all:
     # kafka_connect:
     #   properties:
     #     key: val
+    ## to manage the connector configs from Ansible, set the following:
```

Comment:
This PRs coming along! maybe put this in its own block outside of the properties stuff, as to give it its own section. 
I also don't think the rest_server variable is used anymore. 

Just a thought- we could move it under the kafka_connect group itself. it would look like this
```
kafka_connect:
  vars:
    kafka_connect_connectors:
```
But do include the docs. Right now there are a lot of variables under all.vars, i think the connectors only apply to the connect group. Long term we should probably start making that distinction